src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -40,6 +40,7 @@ static server_entry *_serverEntries = nullptr;
 static sint32 _numServerEntries = 0;
 static std::mutex _mutex;
 static uint32 _numPlayersOnline = 0;
+static auto status_text = STR_SERVER_LIST_CONNECTING;
```

Comment:
Would you be able to switch this line to
`static rct_string_id _statusText = STR_SERVER_LIST_CONNECTING;`
?

Reply:
Okay.

Reply:
You _can_ use auto though.

Reply:
I prefer auto because the compiler already infers the type. But in this case, giving it the explicit type is more consistent with the declarations above, so I get why auto might look a bit weird.

I just got used to languages like Rust where explicitly writing the type is often unnecessary and not idiomatic.